src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -455,7 +455,7 @@ void EnBom_Update(Actor* thisx, GlobalContext* globalCtx) {
     } else {
         thisx->gravity = -1.2f;
         if (this->timer != 0) {
-            if (!this->isPowderKeg || (func_808715B8 == this->actionFunc) || !func_801690CC(globalCtx)) {
+            if (!this->isPowderKeg || (func_808715B8 == this->actionFunc) || !Play_InCsMode(globalCtx)) {
```

Comment:
```suggestion
            if (!this->isPowderKeg || (this->actionFunc == func_808715B8) || !Play_InCsMode(globalCtx)) {
```
if it matches.

Reply:
Just want to double check since it was resolved without a comment, but it doesn't match?

Reply:
I clicked on "Commit suggestion" which resolves the conversation.
This change didn't match, so I reverted it locally, but I forgot to say something about it :sweat_smile: